[TDX-2081]: update developer registration and SSO docs (#3994)
Kong/docs.konghq.com: app/konnect/dev-portal/dev-reg.md

   You should receive an email to confirm your email address. Click the verification link in the email.

4. Set a password for your account.

If your admin has auto-approve enabled, you can log into the Dev Portal immediately after setting a password. Learn how to [set up auto-approval for developer access as an admin](/konnect/dev-portal/access-and-approval/auto-approve-devs-apps/).

If auto-approve is not enabled, an admin has to review and approve your request manually. All manually approved developers are notified via email upon gaining access to the Dev Portal. Learn how to [manually approve developer access as an admin](/konnect/dev-portal/access-and-approval/manage-devs/#approve-dev-reg).

## Login/Register with SSO {#sso}
{:.badge .enterprise}

> {{site.konnect_short_name}} Dev Portal supports single sign-on login and registration. This feature must be configured by the {{site.konnect_short_name}} site admin before being made available on the Dev Portal. For information on how to enable SSO on your Dev Portal as a {{site.konnect_short_name}} admin, review the [enable single sign-on instructions](/konnect/dev-portal/customization/#single-sign-on/).

Reviewer: ```suggestion
The {{site.konnect_short_name}} Dev Portal supports single sign-on (SSO) login and registration. The {{site.konnect_short_name}} Admin must configure SSO before it's available on the Dev Portal. For information on how to enable SSO on your Dev Portal as a {{site.konnect_short_name}} Admin, review the [enable single sign-on instructions](/konnect/dev-portal/customization/#single-sign-on/).
```
Here "single sign-on" isn't capped, but you capped it before. Looks like Google has it in lowercase.

> 1. Navigate to the {{site.konnect_short_name}} Dev Portal URL provided by your {{site.konnect_short_name}} admin.

Reviewer: ```suggestion
1. Navigate to the {{site.konnect_short_name}} Dev Portal URL provided by your {{site.konnect_short_name}} Admin.
```

2. From the login form, click **Login with SSO**.

> 3. You will be redirected to the Identity Provider (IdP) that the {{site.konnect_short_name}} admin has configured. Sign in using your IdP's credentials.

Reviewer: ```suggestion
3. After you are redirected to the Identity Provider (IdP) that the {{site.konnect_short_name}} Admin configured, sign in using your IdP's credentials.
```
Not sure how I feel about the "After you are..." edit that I made, I just couldn't think of another way to get around using future tense there.

Reviewer 2: I like your suggestion.

> 4. Once you've logged in, you will be redirected to the {{site.konnect_short_name}} catalog.

Reviewer: ```suggestion
4. Once you've logged in, you are redirected to the {{site.konnect_short_name}} catalog.
```
